SubZero.pro.user

```diff
@@ -0,0 +1,271 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You don't actually want to be adding the .pro.user file to github because it's personalized for each user.